scipy/interpolate/_rgi.py

```diff
@@ -6,7 +6,7 @@
 
 import scipy.sparse.linalg as ssl
 
-from .interpnd import _ndim_coords_from_arrays
+from ._interpnd import _ndim_coords_from_arrays  # type: ignore[import-not-found]
```

Comment:
Sorry for the two piece review! I think instead of adding the comment to keep mypy happy we can update the following line here https://github.com/scipy/scipy/blob/beb6b2fbe656eae47b117fa8c416cdd88c27c98e/mypy.ini#L125

Reply:
Makes sense, thanks! Done and force-pushed.